typescript/packages/subsurface-viewer/src/layers/grid3d/privateLayer.ts

```diff
@@ -14,16 +13,14 @@ import type {
     ExtendedLayerProps,
     colorMapFunctionType,
 } from "../utils/layerTools";
-
 import { getImageData } from "../utils/layerTools";
-
+import { Texture2D } from "@luma.gl/webgl";
```

Comment:
Do you need this one ?

Reply:
Yes. (Its used on line 196)

Reply:
Sorry, I did oversee that you only moved that line :(